prjxray/db.py

```diff
@@ -25,14 +25,15 @@ def get_available_databases(prjxray_root):
 
 
 class Database(object):
-    def __init__(self, db_root):
```

Comment:
The merge resolution is backwards.  Tools should always specify the db_root, rather than relying on implicit env vars.

Reply:
I disagree. I want convenient tools

Reply:
The alternate solution would be a util class to make the DB. Does that work for you?

Reply:
The tools can default to the env vars, but need to work in the absence of those env vars.  Env vars need to be kept in the roots of tools as much as possible, especially for tools that will be used extensively outside of database generation.

Reply:
I'm fine with a util function to make the DB, but in this case the tool needs to be able to take the db_root from the command line, so the util function is redundant.

Reply:
Ah I see. I misunderstood what the merge conflict was. Let me re-review